app/src/main/kotlin/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -219,6 +217,9 @@ class CarSensorManager :
 
     private lateinit var latestContext: Context
 
+    @Suppress("ObsoleteSdkInt") // Needed for app module (minSdk 21) but obsolete for automotive (minSdk 29)
+    private val areCarSensorApisAvailable get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
```

Comment:
```suggestion
    private val areCarSensorApisAvailable = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
```

I don't know why we have this ugly pattern in this class if it is to get different values over time it shouldn't be an attributes but a function (CC @jpelgrom ). At least for the one you've added should be updated.